blog/ga-for-neon.adoc

```diff
@@ -0,0 +1,101 @@
+= JBoss Tools and Red Hat Developer Studio for Eclipse Neon
+:page-layout: blog
+:page-author: akazakov
+:page-tags: [release, jbosstools, devstudio, jbosscentral]
+:page-date: 2016-06-27
+
+link:/downloads/jbosstools/neon/4.4.0.Final.html[JBoss Tools 4.4] and link:/downloads/devstudio/neon/10.0.0.GA.html[Red Hat JBoss Developer Studio 10.0] for Eclipse Neon are here waiting for you. Check it out!
+
+image::/blog/images/devstudio10.png[]
+
+== Installation
+
+JBoss Developer Studio comes with everything pre-bundled in its installer. Simply download it from our https://www.jboss.org/products/devstudio.html[JBoss Products page] and run it like this:
+ 
+    java -jar jboss-devstudio-<installername>.jar
+
+JBoss Tools or Bring-Your-Own-Eclipse (BYOE) JBoss Developer Studio require a bit more:
+
+This release requires at least Eclipse 4.6 (Neon) but we recommend
+using the latest http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/neon[Eclipse 4.6 Neon JEE Bundle] since then you get most of the dependencies preinstalled. 
+
+Once you have installed Eclipse, you can either find us on the Eclipse Marketplace under "JBoss Tools" or "Red Hat JBoss Developer Studio".
+
+For JBoss Tools, you can also use our update site directly.
+
+    http://download.jboss.org/jbosstools/neon/stable/updates/
+
+== What is new? 
+
+Our main focus for this release was on adoption of Eclipse Neon, improvements for container based development and bug fixing.
+Eclipse Neon itself has a lot of new cool stuff but let me highlight just a few updates in both Eclipse Neon and JBoss Tools plugins that I think are worth mentioning.
+
+=== JavaScript Development Tools 2.0
+
+Red Hat team along with other contributers have been working hard for the last months on restarting https://eclipse.org/webtools/jsdt/[JavaScript Development Tools (JSDT)] project for Eclipse Neon.
+The updated tooling includes EcmaScript 2015 support, JSON editor, Node.js, grunt, gulp, bower and npm tools.
+
+Here is a short https://vimeo.com/167812779[video] which demonstrates new features and enhancements of JavaScript Tools:  
+
+video::167812779[vimeo, width=640, height=400]
+
+=== Improved OpenShift 3 and Docker Tools
+
+We are continue to work on providing better experience for container based development in JBoss Tools and Developer Studio. Let's go through a few interesting updates here and you can find more details on link:/documentation/whatsnew/jbosstools/4.4.0.Final.html[the What's New page].
+
+==== Dockerfile Editor
+
+The new Dockerfile editor provides users with content assist on the commands (ADD, COPY, RUN, etc.) as well as a customizable syntax highlighting.
+
+image::/documentation/whatsnew/docker/images/docker_neon_m7/docker_editor.png[Docker Editor]
+
+==== Deploy docker images to the CDK OpenShift registry
+
+The New OpenShift Connection wizard supports setting a Docker registry url in the advanced properties section.
```

Comment:
Worth adding that the URL will be automatically added when the connection has been created by the CDK server adapter